lib/tipsifyGltf.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+var clone = require('clone');
+var readAccessor = require('./readAccessor');
+var Cesium = require('cesium');
+var Tipsify = Cesium.Tipsify;
+
+module.exports = tipsifyGltf;
+
+function tipsifyGltf(gltf) {
+    var accessors = gltf.accessors;
+    var meshes = gltf.meshes;
+    for (var meshId in meshes) {
+        if (meshes.hasOwnProperty(meshId)) {
+            var mesh = meshes[meshId];
+            var primitives = mesh.primitives;
+            var primitivesLength = primitives.length;
+            for (var i = 0; i < primitivesLength; i++) {
+                var primitive = primitives[i];
+                var indexAccessorId = primitive.indices;
+                var indexAccessor = accessors[indexAccessorId];
+                var indicesArray = readAccessor(gltf, indexAccessor);
+                
+                Tipsify.tipsify({
```

Comment:
I am currently working on a writeAccessor function, but am at a bit of standstill, which may be caused by an initial misunderstanding of my own:

Currently, I am grabbing the indices from each accessor and tipsifying each set of indices relative to only the indices within that accessor--I am fairly certain this is the way we want to do it.

However, I need to eventually write these tipsified indices to the buffer state, but I cannot do that in the same scope as reading in the accessor, because at that scope I am only looking a specific bufferView and not the entire buffer.

The only solution I am seeing is creating some form of symbol table with keys of the initial byte location and values of the bufferView indices arrays and then writing that to a buffer _after the tipsify loop_. That is a somewhat cumbersome solution, though.

In other words, I'd be doing more of a writeBuffer outside of the loop instead of multiple writeAccessors within the loop since I don't know of a way to just change a single bufferView without rewriting the buffer each time.

Am I missing something? Just wanted to make sure before I go ahead with the cumbersome solution.

I hope I am not missing anything super obvious here. Thanks!